ui_framework/components/local_nav/_local_search.scss

```diff
@@ -4,47 +4,50 @@
   height: $localSearchHeight;
 }
 
-.kuiLocalSearchInput {
-  flex: 1 1 100%;
-  padding: 5px 15px;
-  font-size: $globalFontSize;
-  color: $localNavTextColor;
-  background-color: $localSearchBackgroundColor;
-  border: 2px solid $localSearchBackgroundColor;
-  border-bottom-left-radius: 4px;
-  border-top-left-radius: 4px;
-  border-bottom-right-radius: 0;
-  border-top-right-radius: 0;
+  .kuiLocalSearch__icon {
+    position: absolute;
+    top: 0.5em;
+    left: 0.7em;
+    font-size: 1em;
+    color: #ACACAC;
+  }
 
-  &.kuiLocalSearchInput-isInvalid {
-    border-color: $localSearchBorderColor-isInvalid;
+  .kuiLocalSearch__help {
+    position:absolute;
+    padding: 5px 0;
+    right: 0.7em;
+    font-size: 1em;
+    white-space: nowrap;
   }
 
-  @include darkTheme {
-    color: $localNavTextColor--darkTheme;
-    background-color: $localSearchBackgroundColor--darkTheme;
-    border-color: $localSearchBackgroundColor--darkTheme;
+  .kuiLocalSearch__input {
+    flex: 1 1 100%;
+    padding: 5px 13em 5px 28px;
```

Comment:
@cjcenizal not sure if this line is what prompted you to ask about `em`s, but in case you were wondering I used `em`s for the right padding here so that it would still be correct even if the user changes their browser font size. Otherwise the doc link and the query string could overlap. I can add a comment to that effect if you guys think it's confusing.